src/frontend/src/pages/CalendarPage/CalendarComponents/CalendarDayCard.tsx

```diff
@@ -42,31 +43,75 @@ const CalendarDayCard: React.FC<CalendarDayCardProps> = ({ cardDate, events }) =
 
   const EventCard = (event: DesignReview) => {
     const [isSummaryModalOpen, setIsSummaryModalOpen] = useState(false);
-    const name = event.designReviewId;
+    const name = event.wbsName;
     return (
       <>
         <DRCSummaryModal open={isSummaryModalOpen} onHide={() => setIsSummaryModalOpen(false)} designReview={event} />
         <Box marginLeft={0.5} marginBottom={0.5} onClick={() => setIsSummaryModalOpen(true)} sx={{ cursor: 'pointer' }}>
           <Card sx={{ backgroundColor: 'red', borderRadius: 1, minWidth: 140, maxWidth: 140, minHeight: 20, maxHeight: 20 }}>
             <Stack direction="row">
-              {getTeamTypeIcon(event.teamType.teamTypeId)}
-              <Typography marginLeft={0.5} marginBottom={0.3} fontSize={14}>
-                {name + ' ' + meetingStartTimePipe(event.meetingTimes)}
-              </Typography>
+              {getTeamTypeIcon(event.teamType.name)}
+              <DynamicTooltip
+                title={name + (event.meetingTimes.length > 0 ? ' - ' + meetingStartTimePipe(event.meetingTimes) : '')}
```

Comment:
fuck I swear this was fixed in a different PR, but I guess not. The meetingTimes are correspond to each box on the availability view, so Monday 10-11 AM is 0, Monday 11-12 is 1, etc. all the way to 83 for the final hour on Sunday. So, the meetingStartTimePipe needs to be adjusted to % 12 the starting time and add it to 10AM and I think that should work